engine/apps/api/views/alert_group.py

```diff
@@ -90,6 +95,14 @@ class AlertGroupFilter(DateRangeFilterMixin, ModelFieldFilterMixin, filters.Filt
     )
     with_resolution_note = filters.BooleanFilter(method="filter_with_resolution_note")
     mine = filters.BooleanFilter(method="filter_mine")
+    team = filters.ModelMultipleChoiceFilter(
```

Comment:
Here and below I use the same approach we have for public api, but modified to support multi value filters